tests/test_amity.py

```diff
@@ -9,34 +10,37 @@ class TestAmity(unittest.TestCase):
 	def setUp(self):
 		self.amity = Amity()
 	
-	@patch('app.storage')	
-	def test_room_creation(self,test_storage):
-		test_storage.list_of_all_rooms.return_value = ["CAMELOT","NARNIA"]
-		result = Mock()
-		result.room_name.return_value = "HOGWARDS"
-		print test_storage.list_of_all_rooms.return_value
-		self.result = self.amity.createRoom(result.room_name.return_value)
-		print self.amity.room_name
-		print test_storage.list_of_all_rooms.return_value
-
-		if self.result != "Room created successfully!!":
-			self.assertNotEqual(
-				self.amity.room_name, "", msg = "Cannot create empty room")
-
-			self.assertNotIn(
-				self.amity.room_name,
-				test_storage.list_of_all_rooms.return_value,
-				msg = "Room with that name already exist."
-				)
-			print test_storage.list_of_all_rooms.return_value
+	def test_room_creation(self):
+		size = len(Storage.list_of_all_rooms)
+		if self.amity.createRoom("Hogwards") == "Room created successfully!!":
+
+			self.assertEqual(len(Storage.list_of_all_rooms), size+1)
 		else:
-			self.assertIn(
-				self.amity.room_name,
-				test_storage.list_of_all_rooms.return_value,
-				msg = "Room with that name already exist."
-				)	
+		
+			self.assertEqual(len(Storage.list_of_all_rooms), size)
+	
+	def test_room_allocation(self):
+		size = len(Storage.people_info)
+
+		if self.amity.allocateRoom(
```

Comment:
This if statement seems to evade a failure in the function. You are asserting after you have checked that the function is working. It doesn't make sense to test the obvious. The if statement makes it obvious.

```
allocation_status = self.amity.allocateRoom(" Marvin Jones","FELLOW","YES")
self.assertEqual(allocation_status, "Your room allocation has been successfully!!")
self.assertEqual(len(Storage.people_info), size +1)